pypeit/calibrations.py

```diff
@@ -884,15 +884,22 @@ def check_for_calibs(par, fitstbl, raise_error=True):
             PypeIt run.
         raise_error (:obj:`bool`, optional):
             If True, crash out
+        cut_cfg (`numpy.ndarray`_, optional):
+            Also cut on this restricted configuration (mainly for chk_calibs)
```

Comment:
I think you need an empty line before the `Returns` statement.

Reply:
added